doc/rst/tools/mason/mason.rst

```diff
@@ -159,202 +194,364 @@ To try out different values at runtime, pass the values for ``number`` to ``maso
 
 
 Testing your Project
-====================
+~~~~~~~~~~~~~~~~~~~~
 
-Testing a Mason project is easy! Start by adding test files to the ``test/`` directory and specifying
-them in your Mason.toml as follows:
+Mason makes it easy to test projects. To demonstrate how easy it is, the following is
```

Comment:
> Mason makes it easy to test projects. To demonstrate how easy it is

I don't think this adds much. Also, it may make a user feel especially lost if they are not finding it easy as we suggest.